content/rs/administering/designing-production/networking/port-configurations.md

```diff
@@ -27,7 +27,7 @@ update your firewall with the port for that new database endpoint.
 | TCP | 8444, 9080 | For nginx <-> cnm_http/cm traffic (Internal use) |
 | TCP | 9081 | For CRDB management (Internal use) |
 | TCP | 8070, 8071 | For metrics exported and managed by nginx |
-| TCP | 8080, 9443 | REST API traffic, including cluster management and node bootstrap |
+| TCP | 9443, 8080 (Deprecated) | REST API traffic, including cluster management and node bootstrap |
```

Comment:
It's not deprecated, maybe not-recommended

Reply:
@Paz-Yanover  - what would it take to formally deprecate it so state our intention to disable in the future?